node-tests/blueprints/route-test.js

```diff
@@ -0,0 +1,657 @@
+'use strict';
+
+var fs                 = require('fs-extra');
+var path               = require('path');
+var Promise            = require('ember-cli/lib/ext/promise');
```

Comment:
since `ember-cli` is a dev dependency on `ember` this should be okay, but I'm open to better suggestions.